helm_deploy/prisoner-offender-search/templates/full-index-comparison-cronjob.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   labels:
     {{- include "app.labels" . | nindent 4 }}
 spec:
-  schedule: "05 20 * * *"
+  schedule: "05 22 * * *"
```

Comment:
we are getting bed moves at this time of night - hopefully shifting a bit later will prevent the false positives